domains/sono.json

```diff
@@ -8,7 +8,8 @@
             "185.199.108.153",
             "185.199.109.153",
             "185.199.110.153",
-            "185.199.111.153"
+            "185.199.111.153",
+            "76.76.21.21"
```

Comment:
Why do you need both Github Pages and Vercel IP?

Reply:
In my opinion, it might cause problems between the two so I'd recommend sticking to only Vercel's IP